organizers/2018-09-11.html

```diff
@@ -0,0 +1,110 @@
+---
+layout: organizers
+---
+<!-- Syntax reference: https://github.com/hakimel/reveal.js/blob/master/index.html -->
+
+<section>
+    <h1>The London Django Meetup Group</h1>
+    <h2>September 11, 2018</h2>
+    <img data-src="/img/django-logo-negative.png" style="width:40%" alt="Django logo" class="plain"></img>
```

Comment:
HTML5 uses `<img>` without a closing tag, hence github is highlighting these closing tags in red now. Maybe worth fixing across the whole codebase at once

Reply:
Will do in a subsequent PR.